build.gradle.kts

```diff
@@ -366,6 +383,18 @@ tasks {
     password.set(System.getenv("PRIVATE_KEY_PASSWORD"))
   }
 
+  task<Test>("testIntegration") {
+    doFirst {
+      systemProperty("cody-agent.directory", buildCodyDir.parent)
+      require(System.getenv("SOURCEGRAPH_TEST_TOKEN") != null) {
```

Comment:
How do I specify the Sourcegraph URL? To use the Sourcegraph cli `src`, you need to set both `SRC_ENDPOINT` and `SRC_ACCESS_TOKEN`. It could make sense to respect those same environment variables.

Reply:
Thanks, I was not aware of that! I'm going to change that env var to `SRC_ACCESS_TOKEN`.

I'm still trying to figure out why the tests hangs in your case. They should take few seconds max.

Question: Do we currently need to support `SRC_ENDPOINT`? For now, I don't see any test case that would benefit from that. Of course, there is possibility that such variable will be needed future, but handling this address doesn't seem trivial at first glance. I think adding support for such a variable should be part of future PRs that adds tests that may require to customize this address.

Reply:
Using SRC_ACCESS_TOKEN now.